requirements.txt

```diff
@@ -100,14 +108,6 @@ flask-sqlalchemy==2.4.1 \
     # via
     #   flask-migrate
     #   iib (setup.py)
-flask==1.1.1 \
```

Comment:
Why was this changed? Automatically or was this done manually?  It's the exact same as before just in a different position.

Reply:
It is changed because somebody generate this file with a different python version before. You can check line no.2 and it is changed `with python 3.6`

Reply:
this is automatically generated.